src/ui/components/sdcard.c

```diff
@@ -19,10 +19,15 @@
 #include <hardfault.h>
 #include <screen.h>
 #include <sd.h>
-#include <string.h>
 #include <touch/gestures.h>
 #include <ui/screen_stack.h>
 
+// TODO: use a TIMER interrupt to get a more accurate timer.
+// 250 is ~0.5 sec. Unit: rendering rate.
+#define ACTIVE_AFTER 250
+
+static int _count = 0;
```

Comment:
move this to the data_t struct to avoid globals and so it is initialized to 0 when the component is created.